### DISCUSSION TITLE
About loss function questions?

### DISCUSSION THREAD

**Discussion by xuyaojian123** [lsongx/nerfplayer-nerfstudio#11] [Q&A] [answered]
Why regularization loss is to minimize the probability of deform field and new field. The probability of deform file and new field to 0, so how does the scene have any dynamics?

 
![image](https://github.com/lsongx/nerfplayer-nerfstudio/assets/51220835/adf52ac5-c2ea-43ed-849c-b9a346a3e4db)

**Comment by lsongx** [comment, marked as answer]
Hi, if it does not model dynamic objects, the reconstruction loss will be very large. The goal is to minimize a weighted sum of the two losses.
